traceapp/tmpl/doc.go

```diff
@@ -3,4 +3,4 @@
 // This package needs to be go generated after making changes to template files.
 package tmpl
 
-//go:generate go run -tags=dev data_generate.go
+//go:generate vfsgen -tags=dev -source="sourcegraph.com/sourcegraph/appdash/traceapp/tmpl".Data -output-tags=!dev
```

Comment:
I think the main point I was trying to make above is this: specifying the import path, variable name, and build tags is completely correct, explicit, and not magic. Similarly, specifying the output parameters where you're not happy with the default values also seems completely valid and not magical.

Conversely, had vfsgen tried to figure out the source build tags from source via some heuristics, *that* would feel very magical and hard to understand.